src/Backend/Core/Layout/Templates/macros.html.twig

```diff
@@ -8,3 +8,23 @@
 {% macro required() %}
   <abbr data-toggle="tooltip" title="{{ 'lbl.RequiredField'|trans|ucfirst }}">*</abbr>
 {% endmacro %}
+
+{% macro dataGrid(dataGrid, noItemsMessage) %}
+  <div class="row">
+    <div class="col-md-12">
```

Comment:
what is the point of first doing row and then col-md-12?
I think you can leave both out